### PR TITLE
Linux fixes

### PR DESCRIPTION
**Pull request by skmp**
Nearly worked on first try!

Also added a check if run with no args.

### CODE REVIEW

**Comment by georgemoralis** [comment]
if it is from skmp it should be fine :D

File: src/Core/Memory.cpp [hunk 1]
```diff
@@ -7,12 +7,24 @@
 	#include <sys/mman.h>
 #endif
 
+#if !defined(_WIN64)
+enum PosixPageProtection
+{
+	PAGE_NOACCESS = 0,
+    PAGE_READONLY          = PROT_READ,
```
Comment (VelocityRa) [inline src/Core/Memory.cpp:14]: de sou emathe h mama sou na min anakateveis spaces kai tabs

Comment (wheremyfoodat) [reply src/Core/Memory.cpp:14]: clang-format re malakes

Comment (georgemoralis) [reply src/Core/Memory.cpp:14]: σιωπη ρε βλαμενα θα μου διωξετε τον dev

**Pull request merged**